src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -202,9 +202,6 @@
   <data name="Generators" xml:space="preserve">
     <value>Generators</value>
   </data>
-  <data name="GenericPlaceholderTemplateContextError" xml:space="preserve">
```

Comment:
I am not sure about removing items from here. There is localization for all of the old strings (however the localization for `GenericPlaceholderTemplateContextError` should be for the old string that was here) and we currently can not run code with localization. I doubt that having a localization for some string without the corresponding English string may break something (normally it should not), but better to check it or leave this be.

Reply:
This key is not used anywhere else now, I think it is safe to delete it.

Reply:
It is not used anywhere in the English version, but it still presents in [localization files](https://github.com/dotnet/templating/blob/c9456315a6fb1c59b288c4ebef6dbf34063ef1ca/localize/RUS/Microsoft.TemplateEngine.Cli.resources.dll.lcl#L750). It should not break anything normally, I suppose, but still it would be nice to check that.

Reply:
Should we remove it from localization files as well? How the localization files are generated/updated?

Reply:
Well, if we are sure that we are not going to use this string further, I suppose we may safely remove it from localization files. As for localization files themselves, we will need to request for making a localization for us at some point. Do not know the details, though, that is the question for @grinrag.

Reply:
Reverted changes, marked property as "to be removed"